documentation.yml

```diff
@@ -53,3 +53,13 @@ toc:
       controls or other extensions.
   - EventData
   - Evented
+  - name: Custom Sources
+    description: |
+      These interfaces and methods allow you to create and use third-party data
+      sources.  See also [Map#addSourceType](#Map#addSourceType)
+  - Source
+  - WorkerSource
+  - CreateSourceFunction
```

Comment:
Ah -- it is documented [here](https://github.com/mapbox/mapbox-gl-js/pull/2667/files#diff-d5506f072d1241401a7e31ff642c1606R103), but misspelled, so it wouldn't have shown up in a text search.  Anyway, this was for the source factory function, which is going away per [your comment above](https://github.com/mapbox/mapbox-gl-js/pull/2667#discussion_r69032008)